README.md

```diff
@@ -6,6 +6,23 @@
 
 A C++ library with APIs in Python and R for efficient storage and retrieval of single-cell data using [TileDB Embedded][tiledb].
 
+Note this branch, `main-old`, implements the [original specification](https://github.com/single-cell-data/TileDB-SOMA/blob/main-old/spec/specification.md). Please also see the `main` branch which implements the [updated specfication](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).
+
+# Language bindings and installation instructions
+
+* [Python](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/python)
+* [R](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/r)
+* [API documentation](https://tiledb-inc-tiledb-soma.readthedocs-hosted.com/en/latest/)
+
+# Issues and contacts
+
+* Any/all questions, comments, and concerns are welcome at https://github.com/single-cell-data/TileDB-SOMA/issues
+* Bug reports should ideally be accompanied by a minimally reproducible example, using sample data available outside your organization
+
```

Comment:
suggest similar security notification addition that you made to the other README

Reply:
yup, on the way! :)